projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -639,6 +659,55 @@ public function edit_dsp_generic( $path, $req, $args = array() ) {
 		);
 	}
 
+	/**
+	 * Will check the posts for prices and add them to the posts array
+	 *
+	 * @param WP_REST_Request $posts The posts object.
+	 * @return array|WP_Error
+	 */
+	protected function add_prices_in_posts( $posts ) {
+
+		if ( ! function_exists( 'wc_get_product' ) ||
+			! function_exists( 'wc_get_price_decimal_separator' ) ||
+			! function_exists( 'wc_get_price_thousand_separator' ) ||
+			! function_exists( 'wc_get_price_decimals' ) ||
+			! function_exists( 'get_woocommerce_price_format' ) ||
+			! function_exists( 'get_woocommerce_currency_symbol' )
+		) {
+			return $posts;
+		}
+
+		foreach ( $posts as $key => $post ) {
+			$product = wc_get_product( $post['ID'] );
```

Comment:
Maybe play defensive here, and check if isset( $post['ID'] before continuing (bail out early).

Maybe use YODA syntax? :)

```
if ( false !== $product ) {
```

Or even better, positive logic, as product would be object,null or false

```
if ( $product ) {
```

Oh, and I'd remove the empty new lines 693,695 etc

Reply:
fixed